backend/app/worker.py

```diff
@@ -20,6 +20,7 @@ def update_stats():
 
 @dramatiq.actor
 def update():
+    search.delete_all_apps()
```

Comment:
@barthalion this is only temporary, when the search index was reset I will send another commit removing this part again